development/mimir-ingest-storage/docker-compose.jsonnet

```diff
@@ -54,8 +54,6 @@ std.manifestYamlDoc({
         '-ingester.ring.prefix=exclusive-prefix',
         '-ingest-storage.kafka.consume-from-position-at-startup=end',
         '-ingest-storage.kafka.consume-from-timestamp-at-startup=0',
-        '-ingest-storage.kafka.ingestion-concurrency=2',
```

Comment:
What this change intentional, or should we keep it enabled in the local dev env, just changing the config option to `...-max`?

Reply:
that was a mistake. I restored them in this commit 4b64ae6f3ca5f31aa15332d508a069444115fd40